README.md

```diff
@@ -93,18 +93,41 @@ To use the Material theme, import the correspondent file from the `theme/materia
 
 
 ## Running demos and tests in a browser
+Ensure you have [npm](https://www.npmjs.com/) and [Bower](https://bower.io) installed.
 
-1. Fork the `vaadin-element` repository and clone it locally.
+
+Clone the `vaadin-element` repository.
+```bash
+$ git clone https://github.com/vaadin/vaadin-element.git
+```
+
+Change into the `vaadin-element` directory.
+```bash
+$ cd vaadin-element
+```
+
+Install the dependencies.
+```bash
+$ npm install
+$ bower install
+```
+
+Launch a webserver to serve the demo & tests.
+```bash
+$ npm start
+```
+
+View the demos at:
```

Comment:
There's something wrong here now. At least this part is not right:

<img src="https://user-images.githubusercontent.com/75155/54595358-207b7b00-4a3b-11e9-9b30-068a8a193d2f.png" width="400">